packages/rke2-flannel/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://github.com/flannel-io/flannel/releases/download/v0.26.0/flannel.tgz
-packageVersion: 00
+packageVersion: 01
```

Comment:
This should be 02. There was a 01 in the meanwhile

Reply:
You need to rebase in addition to bumping the version